api/web3server.go

```diff
@@ -337,13 +343,12 @@ func (svr *Web3Server) getBlockByNumber(in interface{}) (interface{}, error) {
 	}
 
 	blkMetas, err := svr.coreService.BlockMetas(num, 1)
-	if err != nil {
-		return nil, err
-	}
-	if len(blkMetas) == 0 {
-		return nil, errInvalidBlock
+	if err != nil || len(blkMetas) == 0 {
+		if err != nil {
+			log.L().Debug("err in getBlockByNumber", zap.Error(err))
```

Comment:
why combine again?
`svr.coreService.BlockMetas()` code is not changed, i think only need to add the `log.L().Debug()` line?